requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Neuro SAN - Core dependency for agent session management
-neuro-san==0.6.5
+neuro-san>=0.6.18
 
```

Comment:
We will need the latest neuro-san with async fixes in order for the unit test for the new multiagent_decomposer to work.  It was a really difficult bug to find the root cause of,
but like all good bugs, the fix was one line.